03/app.js

```diff
@@ -1,3 +1,9 @@
 export default function randomNumber(min, max) {
+    if (typeof min !== 'number') {
+        throw new Error ('show error if not a number');
```

Comment:
Raczej komunikat powinien zawierać informacje o błędzie, a nie formę "opisową". Ten moment że nie jest to liczba już w tym momencie istnieje ;)